### ISSUE TITLE
Book 05: Sentence 043

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#11]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/663e764f-d1b6-4cf1-b4ea-30bab69a8a09)

```
# sent_id = Book_05_-_The_Omens.txt:43
# text = Ic macujlli capitulo, vncan mjtoa: in tetzaujtl catca, in jtechpa chiquatli.
# text[norm] = Ic macuilli capítulo, oncan mihtoa: in tetzahuitl catca, in itechpa chicuatli.
# text[orig] = Ic·macujlli·capitulo·,·vncan·mj¶toa·:·intetzaujtl·catca·,·injtechpa·chi¶quatli·.
# text[osp] = Capitulo quinto, del mal aguero: que tomauan del chillido de la lechuza.
# references = A&D/169:1
1	Ic	ic	ADV	_	_	2	_	_	Folio=7v|Paragraph=2|Line=26|Norm=Ic
2	macujlli	macuilli	NUM	_	_	3	_	_	Folio=7v|Paragraph=2|Line=26|Norm=macuilli|Gloss=cinco
3	capitulo	capítulo	NOUN	_	State=Abs	6	_	_	Folio=7v|Paragraph=2|Line=26|Norm=capítulo|Analysed=Yes
4	,	,	PUNCT	_		6	punct	_	Folio=7v|Paragraph=2|Line=26|Norm=,|Analysed=Yes
5	vncan	oncan	ADV	_		6	_	_	Folio=7v|Paragraph=2|Line=26|Norm=oncan|Gloss=aquí|Analysed=Yes
6	mjtoa	ihtoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	0	root	_	Orig=mj¶·toa|Folio=7v,7v|Paragraph=2,2|Line=26,27|Norm=mihtoa|Analysed=Yes
7	:	:	PUNCT	_		9	punct	_	Folio=7v|Paragraph=2|Line=27|Norm=:|Analysed=Yes
8	in	in	SCONJ	_	_	9	mark	_	Orig=intetzaujtl|Folio=7v|Paragraph=2|Line=27|Norm=in
9	tetzaujtl	tetzahuitl	NOUN	_	State=Abs	6	advcl	_	Orig=intetzaujtl|Folio=7v|Paragraph=2|Line=27|Norm=tetzahuitl|Analysed=Yes
10	catca	catqui	VERB	_	Subcat=Intr	9	cop	_	Folio=7v|Paragraph=2|Line=27|Norm=catca
11	,	,	PUNCT	_		13	punct	_	Folio=7v|Paragraph=2|Line=27|Norm=,|Analysed=Yes
12	in	in	PRON	_	_	13	nsubj	_	Orig=injtechpa|Folio=7v|Paragraph=2|Line=27|Norm=in
13	jtechpa	techpa	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Poss	9	acl	_	Orig=injtechpa|Folio=7v|Paragraph=2|Line=27|Norm=itechpa|Analysed=Yes
14	chiquatli	chicuatli	NOUN	_	State=Abs	13	nmod	_	Orig=chi¶·quatli|Folio=7v,7v|Paragraph=2,2|Line=27,28|Norm=chicuatli|Analysed=Yes
15	.	.	PUNCT	_		6	punct	_	Folio=7v|Paragraph=2|Line=28|Norm=.|Analysed=Yes
```

**Comment by ftyers** [comment]
We need to cross-check the annotation of *Ic maculli capítulo* with the other chapter headings.

**Comment by ftyers** [comment]
Notes:
- For *in tetzahuitl catca*, see #11
- For the ordinals, see #14

**Comment by Lguyogiro** [comment]
for the rel between _mjtoa_ and _tetzahuitl_, I think this should be `parataxis` (reported speech) or `csubj`. ("(It) is said that <clause>").

**Comment by ftyers** [comment]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/15b059ba-01f9-4c64-a34e-53b2f3d62967)

I think `csubj` is better here, the *mo-* on *mihtoa* refers to *in tetzahuitl catca* right? 

There is also an issue with the *in itechpa chicuatli*, what is the structure there, I don't think *in* is `nsubj`...

**Comment by Lguyogiro** [comment]
> I think csubj is better here, the mo- on mihtoa refers to in tetzahuitl catca right?

yeah, I mean I think the mo- refers to the clause "it was a tetzahuitl"


> There is also an issue with the in itechpa chicuatli, what is the structure there, I don't think in is nsubj...

I like this analysis (i.e. "the portent, that is about the lechuza":
![image](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/fabd2d5d-5170-4939-8481-6d7a0344e7ce)

**Comment by ftyers** [comment]
This works for me, but we need to be careful with other *in itechpa*s. 

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/28e9c1b4-48a7-446e-9118-16b34e387c37)

Does this look good?

**Comment by Lguyogiro** [comment]
LGTM

**Comment by ftyers** [comment]
Fixed in 311b9a3
